fe/fe-core/src/main/java/com/starrocks/planner/SortNode.java

```diff
@@ -221,6 +270,14 @@ protected String getNodeExplainString(String detailPrefix, TExplainLevel detailL
             output.append(isAsc.next() ? "ASC" : "DESC");
         }
         output.append("\n");
+        if (detailLevel == TExplainLevel.VERBOSE) {
+            if (!buildRuntimeFilters.isEmpty()) {
```

Comment:
How about display the runtime filter information even if it's empty, to make debugging easy?

Reply:
Join Node use the same infomation in plan. I keep the same behaviour